cmd/kube-scheduler/app/options/options.go

```diff
@@ -302,36 +303,40 @@ func createClients(config componentbaseconfig.ClientConnectionConfiguration, mas
 
 	// This creates a client, first loading any specified kubeconfig
 	// file, and then overriding the Master flag, if non-empty.
-	kubeConfig, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+	kubeConfigs, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 		&clientcmd.ClientConfigLoadingRules{ExplicitPath: config.Kubeconfig},
 		&clientcmd.ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: masterOverride}}).ClientConfig()
 	if err != nil {
 		return nil, nil, nil, err
 	}
 
-	kubeConfig.AcceptContentTypes = config.AcceptContentTypes
-	kubeConfig.ContentType = config.ContentType
-	kubeConfig.QPS = config.QPS
-	//TODO make config struct use int instead of int32?
-	kubeConfig.Burst = int(config.Burst)
+	for _, kubeConfig := range kubeConfigs.GetAllConfigs() {
+		kubeConfig.AcceptContentTypes = config.AcceptContentTypes
+		kubeConfig.ContentType = config.ContentType
+		kubeConfig.QPS = config.QPS
```

Comment:
The change is fine. I spent a little more on the QPS and Burst settings.
The default value is 50/100 across different components hard coded. 
I am not sure if the stability tests driven by sonya are affected. It might be tuned based on CPU/memory/network.